diff --git a/lib/es/index.js b/lib/es/index.js
index b854a7f9e0c82f7e538e19abb47e0d50958a63ea..93a8a837a922caf97b9f8acad1f8c42013f07b96 100644
--- a/lib/es/index.js
+++ b/lib/es/index.js
@@ -13600,7 +13600,7 @@ class fy {
   constructor() {
     k(this, "isReady", !1);
     k(this, "_fontBook", /* @__PURE__ */ new Map());
-    this._loadFontsToBook();
+    // this._loadFontsToBook();
   }
   async _loadFontsToBook() {
     if (!(this.isReady || !await py()) && "queryLocalFonts" in window)
